### PR TITLE
show maintenance shutdown message after completed

### PR DESCRIPTION
**Pull request by yanganto**
remove the redundant `Option<>` wrapper on the parameter, and use the default value.

### CODE REVIEW

File: tests/test_maintenance_shutdown.py [hunk 1]
```diff
@@ -121,12 +121,8 @@ def test_maintenance_shutdown_metrics(
 
         proc = leader.execute_command(
             "maintenance-shutdown",
+            "--wait",
             "1",  # Use one block window for maintenance shutdown in test
         )
         assert proc.returncode == 0
```
Comment (Mic92) [inline tests/test_maintenance_shutdown.py:127]: Maybe check the stdout here for the message "maintenance shutdown complete"?

Comment (Mic92) [inline tests/test_maintenance_shutdown.py:127]:
```suggestion
        assert proc.stdout == "maintenance shutdown complete\n"
```

**Review by Mic92 — APPROVED** [review]

**Comment by yanganto** [comment]
bors merge

**Comment by yanganto** [comment]
bors merge

**Pull request merged**